linode/instanceconfig/schema_resource.go

```diff
@@ -18,16 +18,27 @@ var resourceSchema = map[string]*schema.Schema{
 	"label": {
 		Type:        schema.TypeString,
 		Required:    true,
-		Description: "The Config’s label for display purposes only.",
+		Description: "The Config's label for display purposes only.",
+	},
+
+	"device": {
+		Type:          schema.TypeList,
+		Elem:          &schema.Resource{Schema: deviceV2Schema},
+		Optional:      true,
+		Computed:      true,
+		ConflictsWith: []string{"devices"},
+		Description:   "A dictionary of device disks to use as a device map in a Linode's configuration profile.",
 	},
 
 	"devices": {
-		Type:        schema.TypeList,
-		Elem:        &schema.Resource{Schema: devicesSchema},
-		Optional:    true,
-		Computed:    true,
-		MaxItems:    1,
-		Description: "A dictionary of device disks to use as a device map in a Linode’s configuration profile.",
+		Type:          schema.TypeList,
+		Elem:          &schema.Resource{Schema: devicesSchema},
+		Optional:      true,
+		Computed:      true,
+		MaxItems:      1,
+		ConflictsWith: []string{"device"},
+		Deprecated:    "Devices attribute is deprecated in flavor of `devices_map` or `device`.",
```

Comment:
I think we can drop `device_map` here :+1:

Reply:
Ah, good catch!

Reply:
Fix incoming, thank you!